src/ui/Roster.ts

```diff
@@ -127,6 +220,21 @@ export default class Roster {
       }
 
       rosterItemElement.remove();
+
+      let contactType = contact.getType();
+      let list = contactType=="chat"?this.contactList:this.groupList;
```

Comment:
Code style should be enforced.

Reply:
now using let list = (contactType==='chat'?this.contactList:this.groupList);